config/default.yaml

```diff
@@ -22,13 +22,14 @@ security:
   cors_origin_methods: "GET,POST,OPTIONS,PUT,DELETE"
   cors_origin_whitelist: "http://localhost:3451,http://localhost:3452"
   secure_auth_cookie_enabled: false
```

Comment:
timing-allow-origin: "A,B,C" is the only new value introduced.  the rest should not be merged